<span style="font-size:26pt"><b>Mapped Faunal Relations Among Mammal Faunas, Based on an Entropy-Maximization Approach </b></span>\n\n<b>Abstract</b>\n\nFor a recent publication the authors identified a seven-region model of mammal family distribution patterns in which each unit contributes equally to the system’s overall statistical characteristics of diversity, without their being measurably equivalent as individual units.  That systemization presents a highly efficient phenetic model, and possibly can be interpreted as a form of natural classification. One of the logical applications of this approach is to areal representation; statistical moments of the underlying world patterns can be used to characterize faunal status at any individual location by relating the latter to the former. For example, a “mean cosmopolitanism” statistic can be derived for an individual location by noting the mean number of regions worldwide in which the elements native to that location are found. In a parallel fashion, comparisons can be made between locations, including in a manner characterizing regional transition zones and their abruptness. An arbitrarily bounded area, the conterminous United States, is treated in this fashion by way of example. \n\n<b>Keywords  </b>mammals, faunal regions, classification, natural systems, evolution, distribution patterns, maps\n\n\n\n<span style="font-size:22pt"><b>1. Introduction</b></span>\n\n\n\nIn a recent study, Smith et al. [1] described a new mammal faunas regionalization scheme based on the data of Holt et al. [2] and an analysis format first employed in 1983 by CHS [3, 4] focusing on family-level patterns. The 1983 papers featured geographic entropy maximization methods [5] and a new argument for understanding relative regional equivalencies; of these papers Whittaker <i>et al.</i> [6, p. 2210], <span style="color:000000">in introducing a special issue on regionalization systems, commented: </span>\n\n<span style="color:000000;font-size:20pt">…</span><span style="font-size:20pt">Smith undertook a pioneering analysis designed to assess the efficiency of this system and whether it provided a logical set of relationships among the subregions, </span><span style="font-size:20pt"><i>i.e.</i></span><span style="font-size:20pt"> how well they nest within the higher level regions. … Smith wrote an innovative companion paper which set out to assess the proposition that evolution might be regarded as a stochastic spatial process with inter-regional affinities explicable through chance interactions between the subregions and a deterministic distance-decay effect on the diffusion of evolutionary innovations. </span>\n\nIn 2008 Wilson [7, p. 868] specifically cited the 1983 studies in a study of how Boltzmann-type statistical physics models might be applied to complex systems by noting: “<span style="color:000000">using the Boltzmann methods … this gives a new ecological model. There is only one empirical example in the literature known to the author [3, 4]. There are two elements of knowledge transfer from urban to ecological modelling here: first, an explicit and potentially more effective way of handling spatial interaction and, second, the ways of analysing the dynamics of spatial patterns.”</span>\n\nThe 2023 work [1] extended this basically phenetic effort by suggesting the possibility that the results could also be interpreted within a framework of “natural system” thinking. The result was a seven-region system that met the earlier criteria, while also conforming to a concept of natural system organization derived from the philosophy of Spinoza [8, 9].  A third paper in 1983 by CHS [10] applied second-order statistics (largely, statistical moments) derived from the first two to produce maps of worldwide mammalian faunal relations; for example, illustrating spatially varying mean and total cosmopolitanism characteristics.  The present work is in the same general form as the third one from 1983, but provides some new examples derived from the data of [1], and an extended discussion.\n\nThe new mammal faunas regionalization presented in [1], including a “Marine” region, is shown in Figure 1.\n\n----Image alt text---->Map\n\nDescription automatically generated<----media/image1.jpeg----\n\n<b>Figure 1</b>  The seven-region mammal families faunal model established in [1] (in part after the data of Holt <i>et al.</i> [2, Figure S4C]).\n\n\n\n<b>2.  The Problem of the Equivalence of Regions</b>\n\nHolt <i>et al.</i>’s 2013 study [2], integrating phylogenetic considerations into the spatial clustering algorithm, produced a hierarchical inclusion tree starting with 36 initial mammalian spatial units. Figure 1 above depicts in spatial form the sixth level of inclusion in that tree hierarchy (with the exception of an additional “‘Marine” Region; their work only dealt with terrestrial forms).  Holt <i>et al.</i>’s work left unanswered the question of what number of regions might most efficiently represent biogeographic regionalization patterns in the Mammalia, probably because of “hierarchy bias”; that is, a sense that a tree-organized arrangement of geographical inclusiveness not strictly mirroring cladistic systematics is “unnatural” to begin with (and thus that there is little profit in trying to identify a single preferred level of regional partitioning). But beyond the cladism assumption, there is a further bias stemming from a more than century’s worth of discussion on the fact that calculated regional faunas, however formulated, are never structurally “equivalent”: some regions will always host more forms than others, and/or have higher or lower numbers of endemic taxa.  For example, the Wallacean/Sclaterian Palaearctic and Nearctic regions as compared to, say, the same scheme’s African and South American regions, have much different diversities and endemism rates. <span style="color:000000">But, and to repeat the question posed in [1], should we even assume that in a probabilistically-evolving system the diversification principles involved would likely, or even could, lead to “faunally equivalent” components?</span>\n\n<span style="color:000000">Smith’s 1983 papers [3, 4] exposed this “hierarchy bias” by showing how a particularly efficient n-region classification could involve regional structures:</span>\n\n<span style="color:000000;font-size:20pt">…whose individual diversity characteristics were equally well specified through a combination of entropy maximization and multiple regression operations based on system-level, rather than individual regions-based, statistics.  The ten-region classification in fact produced very high levels of internal specificity independent of levels of endemism or taxon richness in its individual units.  The implication is that a macroevolution process taking place on a limited planetary surface produces long-term results which, though consistent with a historical process of species lines divergences, nevertheless emerges probabilistically in response to very complex long-term interactions among biological/environmental/geologic constraints and opportunities. [1, p. 2 of 15].</span>\n\n<span style="color:000000">The seven-region model developed in [1] is shown to be self-specifying at even considerably higher levels of significance than the model developed in 1983, suggesting a stochastic process leading to regional units of greatly varying family richness and number of endemic forms. These are yet “equivalent” to the extent that organized as they are they define a system of higher information content. This leads to the suggestion that the seven-region system presented there may not merely be a statistical optimization, but reflective of a natural and recurring pattern of internal differentiation that falls out in a manner observing certain principles of combinatorial likelihood, and/or negentropy. Further treatment of the underlying model may be found in [1] and online at: http://people.wku.edu/charles.smith/once/writings.htm#2; additional new discussion is provided in the final section of the present work.</span>\n\n\n\n<span style="color:000000"><b>3. Cartographic Applications</b></span>\n\n<span style="color:000000">Whether one takes the model presented in [1] to reflect a normative, natural, process, or instead just a highly efficient phenetic representation, the results can be used to illustrate a form of areal representation, first described in the third 1983 paper, which appeared in the Leigh Van Valen-edited journal </span><span style="color:000000"><i>Evolutionary Theory</i></span><span style="color:000000">. In that work, CHS wrote [10, pp. </span>226–227] how an efficient faunal systemization might:\n\n\t\t\t\t\t<span style="font-size:20pt">…</span><span style="color:010101;font-size:20pt">be used to characterize the fauna of any particular location on the earth’s surface by first giving attention to the total number of regions that each faunal element found at the given location is present in on a worldwide basis. A distribution of relational values (one for each form specific to the location and ranging in possible magnitude from one to a maximum equal to the number of regions in the classification) can be tallied from which conventional descriptive statistics may be extracted. . . . This approach to data representation might be termed ‘second-order’ by analogy to the second-order (nearest-neighbor) analysis of point patterns; the major difference lies in the fact that a distribution of relative ‘presences’ is obtained rather than a distribution of distances. In the ‘second-order’ representations of organic distribution described here, therefore, attention is focused not on the attributes of any one, pair of, or entire set of locations, but instead on partial orderings of system-level distributional attributes.</span>\n\n\t\t\t\t\t<span style="color:010101">To illustrate, map figures based on the 1983 mammal regions classification [3] were constructed in [8] that highlighted the conceptual appeal of graphically representing “second-order” statistics emerging from the ten-region classification.  The mammal families present at 504 worldwide locations were recorded, and the number of the ten regions each existed in, noted.  From such tallies one could, at each of the 504 locations, calculate a “mean relative cosmopolitanism” statistic that expressed the degree to which those locations were dominated by wide- or limited-ranging forms. (See [10], Figure 2.)  Local values at the high end topped out at 6.5+ (in the Arctic), and bottomed out at about 3.5 (in Australia), exposing a domination by generalist families in the former, and by endemic forms in the latter.  Another map displayed worldwide “spatial variation in the degree of influence of all specific mammal faunal groupings on the make-up of the faunas at particular locations” [10, Figure 4].  Each map relied on the use of isopleths to portray variation in the values obtained.</span>\n\n\t\t\t\t\t<span style="color:010101">In [10] the examples given focused on the conditions at a sample of worldwide point locations as they related to the overall world standard.  But another advantage of the “second-order” statistics approach is its ability to relate the conditions at any one location not only to the whole, but to (1) any other single location, leading to possible portrayals of tracks and corridors, or (2) the placement and strength of transitions between regions. Some examples follow; these are based on an arbitrarily-chosen study area, the conterminous United States.</span>\n\nWe begin by assuming the present classification model, wherein the world’s mammal faunal patterns have been arranged into exactly 7 geographical regions, <i>r</i>, i.e., <i>r</i> = 7.  In the notation below, an index <i>r=7</i> is kept in the statistics to distinguish this specific model from the more general model established in [1] where the <i>r</i>-region world is possibly divided into 5 to 9 geographical areas.  Each geographical region in any such 7-region model will be referred to with an index <i>i</i> = 1, 2,… to 7.  Mammal families are indexed from <i>f</i> = 1 to <i>N</i> where <i>N</i> is the total number of mammal families (or ‘forms’) in the world (140, after [11]), as applied here. Now, let us denote:\n\n<span style="color:000000;font-size:20pt"><i>d</i></span><span style="color:000000;font-size:20pt"><i><sub>f,i,r=7</sub></i></span><span style="color:000000;font-size:20pt"> for </span><span style="color:000000;font-size:20pt"><i>f</i></span><span style="color:000000;font-size:20pt"> = 1,…</span><span style="color:000000;font-size:20pt"><i>N</i></span><span style="color:000000;font-size:20pt">, </span><span style="color:000000;font-size:20pt"><i>i</i></span><span style="color:000000;font-size:20pt"> = 1,...,7;  a dummy variable that takes a value of 1 if there is such a mammal family </span><span style="color:000000;font-size:20pt"><i>f</i></span><span style="color:000000;font-size:20pt"> in region </span><span style="color:000000;font-size:20pt"><i>i</i></span><span style="color:000000;font-size:20pt"> of the </span><span style="color:000000;font-size:20pt"><i>7</i></span><span style="color:000000;font-size:20pt">-region model, and 0 otherwise. The collection of </span><span style="color:000000;font-size:20pt"><i>d</i></span><span style="color:000000;font-size:20pt"><i><sub>f,i,r=7</sub></i></span><span style="color:000000;font-size:20pt"><sub> </sub></span><span style="color:000000;font-size:20pt"> forms a Table of presence (1) and  absence (0) of mammal families across all regions </span><span style="color:000000;font-size:20pt"><i>i</i></span><span style="color:000000;font-size:20pt"> (see Sheet 1 of the supplemental data appendix.   </span>\n\n<span style="color:000000;font-size:20pt"><i>div</i></span><span style="color:000000;font-size:20pt"><i><sub>i,r=7</sub></i></span><span style="color:000000;font-size:20pt">, </span><span style="color:000000;font-size:20pt"><i>i</i></span><span style="color:000000;font-size:20pt"> = 1,...,</span><span style="color:000000;font-size:20pt"><i>7</i></span><span style="color:000000;font-size:20pt">; the diversity (or richness) of each region </span><span style="color:000000;font-size:20pt"><i>i of </i></span><span style="color:000000;font-size:20pt">the 7-region model, computed as the count of families found therein, with </span><latex>divi,r=7=f=1Ndf,i,r=7</latex><span style="color:000000;font-size:20pt"> (see Sheet 1, supplemental data appendix).</span>\n\n<span style="color:000000;font-size:20pt"><i>cw</i></span><span style="color:000000;font-size:20pt"><i><sub>f,i,r=7 </sub></i></span><span style="color:000000;font-size:20pt">, </span><span style="color:000000;font-size:20pt"><i>f </i></span><span style="color:000000;font-size:20pt">= 1,…,N, </span><span style="color:000000;font-size:20pt"><i>i </i></span><span style="color:000000;font-size:20pt">=1…,7; the cosmopolitanism weight indicating whether a family </span><span style="color:000000;font-size:20pt"><i>f</i></span><span style="color:000000;font-size:20pt"> found in region </span><span style="color:000000;font-size:20pt"><i>i</i></span><span style="color:000000;font-size:20pt"> is endemic to the region or cosmopolitan (i.e., is found in a number of other regions of the 7-region world). For example, the cosmopolitanism weight of family </span><span style="color:000000;font-size:20pt"><i>f</i></span><span style="color:000000;font-size:20pt"> of region </span><span style="color:000000;font-size:20pt"><i>i</i></span><span style="color:000000;font-size:20pt"> =1 in our 7-region world is given by: </span><latex>cwf,i=1,r=7=(df,i=1,r=7+i≠17df,i,r=7)</latex><span style="color:000000;font-size:20pt"> if </span><latex>df,i=1,r=7=1</latex><span style="color:000000;font-size:20pt">, else </span><latex>cwf,i=1,r=7=0</latex><span style="color:000000;font-size:20pt"> (see Sheet 2, supplemental data appendix).</span>\n\n<span style="color:000000;font-size:20pt"><i>z</i></span><span style="color:000000;font-size:20pt"><i><sub>i,r=7</sub></i></span><span style="color:000000;font-size:20pt">, </span><span style="color:000000;font-size:20pt"><i>i</i></span><span style="color:000000;font-size:20pt"> = 1,...,</span><span style="color:000000;font-size:20pt"><i>7</i></span><span style="color:000000;font-size:20pt">; the “zeroth moment” (or “total cosmopolitanism”) of a given region </span><span style="color:000000;font-size:20pt"><i>i</i></span><span style="color:000000;font-size:20pt"> with </span><latex>zi,r=7=f=1Ncwf,i,r=7</latex><span style="color:000000;font-size:20pt"> (see Sheet 2, supplemental data appendix).</span>\n\n<span style="color:000000;font-size:20pt"><i>z</i></span><span style="color:000000;font-size:20pt"><i><sub>i,r=7</sub></i></span><span style="color:000000;font-size:20pt"><i>/div</i></span><span style="color:000000;font-size:20pt"><i><sub>i,r=7</sub></i></span><span style="color:000000;font-size:20pt">  = the “mean cosmopolitanism" for a region </span><span style="color:000000;font-size:20pt"><i>i</i></span><span style="color:000000;font-size:20pt"> (see Sheet 2, supplemental data appendix).</span>\n\n<span style="color:000000;font-size:20pt"><i>nz</i></span><span style="color:000000;font-size:20pt"><i><sub>i,r=7</sub></i></span><span style="color:000000;font-size:20pt">, </span><span style="color:000000;font-size:20pt"><i>i</i></span><span style="color:000000;font-size:20pt"> = 1,...,</span><span style="color:000000;font-size:20pt"><i>7</i></span><span style="color:000000;font-size:20pt">; the “negative zeroth moment,” in other words, the complementary value to the zeroth moment and defined as: </span><latex>divi,r=7×7-zi,r=7</latex><span style="color:000000;font-size:20pt"> where the first term represents the maximum ‘potential’ total cosmopolitanism of a region </span><span style="color:000000;font-size:20pt"><i>i</i></span><span style="color:000000;font-size:20pt"> with diversity </span><span style="color:000000;font-size:20pt"><i>div</i></span><span style="color:000000;font-size:20pt"><i><sub>i,r=7</sub></i></span><span style="color:000000;font-size:20pt"> (see Sheet 2, supplemental data appendix).</span>\n\n<span style="color:000000;font-size:20pt">The statistics defined above characterize some aspects of a single region </span><span style="color:000000;font-size:20pt"><i>i</i></span><span style="color:000000;font-size:20pt"> of the world. Statistics pertaining to the analysis of a pair of regions </span><span style="color:000000;font-size:20pt"><i>i,j,</i></span><span style="color:000000;font-size:20pt">=1…,7, can also be defined as follows: </span>\n\n<span style="color:000000;font-size:20pt"><i>cw</i></span><span style="color:000000;font-size:20pt"><i><sub>f,i,j,r=7 </sub></i></span><span style="color:000000;font-size:20pt">, </span><span style="color:000000;font-size:20pt"><i>f </i></span><span style="color:000000;font-size:20pt">= 1,…,N, </span><span style="color:000000;font-size:20pt"><i>i,j, </i></span><span style="color:000000;font-size:20pt">=1…,7; the cosmopolitanism weight of a pair of regions indicating whether a family </span><span style="color:000000;font-size:20pt"><i>f</i></span><span style="color:000000;font-size:20pt"> found in both regions </span><span style="color:000000;font-size:20pt"><i>i</i></span><span style="color:000000;font-size:20pt"> and </span><span style="color:000000;font-size:20pt"><i>j</i></span><span style="color:000000;font-size:20pt"> is also found in other regions of the 7-region world with: </span><latex>cwf,i=1,i=2,r=7=(df,i=1,r=7+df,i=2,r=7+i≠1,27df,i,r=7)</latex><span style="color:000000;font-size:20pt"> if </span><latex>df,i=1,r=7=df,i=2,r=7=1</latex><span style="color:000000;font-size:20pt">, else </span><latex>cwf,i=1,i=2,r=7=0</latex><span style="color:000000;font-size:20pt"> (see Sheet 3, supplemental data appendix).</span>\n\n<span style="color:000000;font-size:20pt"><i>z</i></span><span style="color:000000;font-size:20pt"><i><sub>i,j,r=7</sub></i></span><span style="color:000000;font-size:20pt">, </span><span style="color:000000;font-size:20pt"><i>i,j,</i></span><span style="color:000000;font-size:20pt"> = 1,...,</span><span style="color:000000;font-size:20pt"><i>7</i></span><span style="color:000000;font-size:20pt">, the “zeroth moment” (or “total cosmopolitanism”) of a pair of regions </span><span style="color:000000;font-size:20pt"><i>i,j</i></span><span style="color:000000;font-size:20pt">. For example, </span><latex>zi=1,i=2,r=7=f=1Ncwf,i=1,i=2,r=7</latex><span style="color:000000;font-size:20pt"> when </span><latex>df,i=1,r=7and df,i=2,r=7=1 </latex><span style="color:000000;font-size:20pt">and 0 otherwise (see Sheet 3, supplemental data appendix).</span>\n\n<span style="color:000000;font-size:20pt"><i>nz</i></span><span style="color:000000;font-size:20pt"><i><sub>i,j,r=7</sub></i></span><span style="color:000000;font-size:20pt">, </span><span style="color:000000;font-size:20pt"><i>i,j,</i></span><span style="color:000000;font-size:20pt"> = 1,...,</span><span style="color:000000;font-size:20pt"><i>7</i></span><span style="color:000000;font-size:20pt">; the “negative zeroth moment,” of a pair of region </span><span style="color:000000;font-size:20pt"><i>i,j</i></span><span style="color:000000;font-size:20pt">, with </span><latex>nzi,j,r=7=(ci,j,r=7×7)-zi,j,r=7</latex><span style="color:000000;font-size:20pt">, where </span><span style="color:000000;font-size:20pt"><i>c</i></span><span style="color:000000;font-size:20pt"><i><sub>i,j,r=7 </sub></i></span><span style="color:000000;font-size:20pt">represents the number of common families between two regions </span><span style="color:000000;font-size:20pt"><i>i</i></span><span style="color:000000;font-size:20pt"> and </span><span style="color:000000;font-size:20pt"><i>j</i></span><span style="color:000000;font-size:20pt"> in a world of </span><span style="color:000000;font-size:20pt"><i>7</i></span><span style="color:000000;font-size:20pt"> regions (see Sheets 3 and 4, respectively for construction of </span><span style="color:000000;font-size:20pt"><i>nz</i></span><span style="color:000000;font-size:20pt"><i><sub>i,j,r=7</sub></i></span><span style="color:000000;font-size:20pt"> and </span><span style="color:000000;font-size:20pt"><i>c</i></span><span style="color:000000;font-size:20pt"><i><sub>i,j,r=7</sub></i></span><span style="color:000000;font-size:20pt">). Note that </span><span style="color:000000;font-size:20pt"><i>nz</i></span><span style="color:000000;font-size:20pt"><i><sub>i,j,r=7</sub></i></span><span style="color:000000;font-size:20pt"> collapses to </span><latex>nzi,i,r=7=(ci,i,r=7×7)-zi,i,r=7=(divi,r=7×7)-zi,r=7</latex><span style="color:000000;font-size:20pt"> when </span><span style="color:000000;font-size:20pt"><i>i=j</i></span><span style="color:000000;font-size:20pt">, as the common families between a region </span><span style="color:000000;font-size:20pt"><i>i</i></span><span style="color:000000;font-size:20pt"> and itself is indeed the total number of families in that region (i.e., its diversity). </span>\n\n<span style="color:000000;font-size:20pt"><i>dis</i></span><span style="color:000000;font-size:20pt"><i><sub>i,j,r=7</sub></i></span><span style="color:000000;font-size:20pt"><sub>, </sub></span><span style="color:000000;font-size:20pt"><i>i</i></span><span style="color:000000;font-size:20pt">,</span><span style="color:000000;font-size:20pt"><i>j</i></span><span style="color:000000;font-size:20pt"> = 1,...,</span><span style="color:000000;font-size:20pt"><i>7</i></span><span style="color:000000;font-size:20pt">;  the number of distinct families between two regions </span><span style="color:000000;font-size:20pt"><i>i</i></span><span style="color:000000;font-size:20pt"> and </span><span style="color:000000;font-size:20pt"><i>j</i></span><span style="color:000000;font-size:20pt"> in a world of </span><span style="color:000000;font-size:20pt"><i>7</i></span><span style="color:000000;font-size:20pt"> regions. In particular </span><span style="color:000000;font-size:20pt"><i>dis</i></span><span style="color:000000;font-size:20pt"><i><sub>i,j,r=7</sub></i></span><span style="color:000000;font-size:20pt"><sub>,</sub></span><span style="color:000000;font-size:20pt"> is the number of families found in </span><span style="color:000000;font-size:20pt"><i>i</i></span><span style="color:000000;font-size:20pt"> but not in </span><span style="color:000000;font-size:20pt"><i>j</i></span><span style="color:000000;font-size:20pt"> while </span><span style="color:000000;font-size:20pt"><i>dis</i></span><span style="color:000000;font-size:20pt"><i><sub>j,i,r=7</sub></i></span><span style="color:000000;font-size:20pt"> is the number of families found in </span><span style="color:000000;font-size:20pt"><i>j</i></span><span style="color:000000;font-size:20pt"> but not in </span><span style="color:000000;font-size:20pt"><i>i</i></span><span style="color:000000;font-size:20pt"> (see Sheet 5, supplemental data appendix).</span>\n\n<span style="color:000000;font-size:20pt"><i>s</i></span><span style="color:000000;font-size:20pt"><i><sub>i,j,r=7</sub></i></span><span style="color:000000;font-size:20pt">, </span><span style="color:000000;font-size:20pt"><i>i</i></span><span style="color:000000;font-size:20pt"> and </span><span style="color:000000;font-size:20pt"><i>j</i></span><span style="color:000000;font-size:20pt"> = 1,...,</span><span style="color:000000;font-size:20pt"><i>7</i></span><span style="color:000000;font-size:20pt">; the Smith similarity index between two regions, </span><span style="color:000000;font-size:20pt"><i>i</i></span><span style="color:000000;font-size:20pt"> and </span><span style="color:000000;font-size:20pt"><i>j</i></span><span style="color:000000;font-size:20pt">, in a world of </span><span style="color:000000;font-size:20pt"><i>7</i></span><span style="color:000000;font-size:20pt"> regions with </span><span style="color:000000;font-size:20pt"><i>s</i></span><span style="color:000000;font-size:20pt"><i><sub>i,j,r=7</sub></i></span><span style="color:000000;font-size:20pt"><i> = c</i></span><span style="color:000000;font-size:20pt"><i><sub>i,j,r=7</sub></i></span><span style="color:000000;font-size:20pt"><sub> </sub></span><span style="color:000000;font-size:20pt">- (</span><span style="color:000000;font-size:20pt"><i>dis</i></span><span style="color:000000;font-size:20pt"><i><sub>i,j,r=7</sub></i></span><span style="color:000000;font-size:20pt"><i> + dis</i></span><span style="color:000000;font-size:20pt"><i><sub>j,i,r=7</sub></i></span><span style="color:000000;font-size:20pt">) or in a short-cut notation: </span><span style="color:000000;font-size:20pt"><i>s</i></span><span style="color:000000;font-size:20pt"><i><sub>i,j,r=7 </sub></i></span><span style="color:000000;font-size:20pt">= </span><span style="color:000000;font-size:20pt"><i>a</i></span><span style="color:000000;font-size:20pt"> – (</span><span style="color:000000;font-size:20pt"><i>b</i></span><span style="color:000000;font-size:20pt"> + </span><span style="color:000000;font-size:20pt"><i>c</i></span><span style="color:000000;font-size:20pt">), where ‘</span><span style="color:000000;font-size:20pt"><i>a</i></span><span style="color:000000;font-size:20pt">’ equals the number of families common to the regional pairing in question, and from which we subtract the number of families found in </span><span style="color:000000;font-size:20pt"><i>i</i></span><span style="color:000000;font-size:20pt"> but not in </span><span style="color:000000;font-size:20pt"><i>j</i></span><span style="color:000000;font-size:20pt"> (‘</span><span style="color:000000;font-size:20pt"><i>b</i></span><span style="color:000000;font-size:20pt">’ = </span><span style="color:000000;font-size:20pt"><i>dis</i></span><span style="color:000000;font-size:20pt"><i><sub>i,j,r=7</sub></i></span><span style="color:000000;font-size:20pt">), and the number of families found in </span><span style="color:000000;font-size:20pt"><i>j</i></span><span style="color:000000;font-size:20pt"> but not in </span><span style="color:000000;font-size:20pt"><i>i</i></span><span style="color:000000;font-size:20pt"> (‘</span><span style="color:000000;font-size:20pt"><i>c</i></span><span style="color:000000;font-size:20pt">’ =</span><span style="color:000000;font-size:20pt"><i>dis</i></span><span style="color:000000;font-size:20pt"><i><sub>j,i,r=7</sub></i></span><span style="color:000000;font-size:20pt">) (see Sheet 5, supplemental data appendix).</span>\n\n\t\t\t\t\t<span style="color:010101">In our specific example, a regular sample grid of 295 cells was set across an equal-area map of the lower forty-eight (conterminous) states of the United States (Figure 2); the number of mammal families (excluding strictly marine forms) present in each was then tallied, based on a large number (many dozens) of sources. </span><span style="color:000000">Each cell will be referred with index </span><span style="color:000000"><i>c</i></span><span style="color:000000"> = 1, 2,… 295.  Mammal families in the USA are indexed as </span><span style="color:000000"><i>f</i></span><span style="color:000000"> = 1,…to </span><span style="color:000000"><i>N</i></span><span style="color:000000"><i><sub>US</sub></i></span><span style="color:000000"> where </span><span style="color:000000"><i>N</i></span><span style="color:000000"><i><sub>US</sub></i></span><span style="color:000000"> is the total number of mammal families (32) recorded in the USA.  We can now define some statistics for each of these USA sample cells.  For convenience and to distinguish this model from the 7-region model above, a </span><span style="color:000000"><i>usa</i></span><span style="color:000000">-subscript has been added in the notation below. Let us assume:</span>\n\n<span style="color:000000;font-size:20pt"><i>d</i></span><span style="color:000000;font-size:20pt"><i><sub>f,c,usa</sub></i></span><span style="color:000000;font-size:20pt"> for </span><span style="color:000000;font-size:20pt"><i>f</i></span><span style="color:000000;font-size:20pt"> = 1,…,</span><span style="color:000000;font-size:20pt"><i>N</i></span><span style="color:000000;font-size:20pt"><i><sub>US</sub></i></span><span style="color:000000;font-size:20pt">, </span><span style="color:000000;font-size:20pt"><i>c</i></span><span style="color:000000;font-size:20pt"> = 1,...295; a dummy variable that takes a value of 1 if there is such a family </span><span style="color:000000;font-size:20pt"><i>f</i></span><span style="color:000000;font-size:20pt"> in cell </span><span style="color:000000;font-size:20pt"><i>c</i></span><span style="color:000000;font-size:20pt"> , and 0 otherwise. The collection of </span><span style="color:000000;font-size:20pt"><i>d</i></span><span style="color:000000;font-size:20pt"><i><sub>f,c,usa</sub></i></span><span style="color:000000;font-size:20pt"><sub> </sub></span><span style="color:000000;font-size:20pt">forms the Table of presence (1) and absence (0) of mammal families across all cells in the USA (see Sheet 7, supplemental data appendix).</span>\n\n<span style="color:000000;font-size:20pt"><i>div</i></span><span style="color:000000;font-size:20pt"><i><sub>c,usa</sub></i></span><span style="color:000000;font-size:20pt"> for </span><span style="color:000000;font-size:20pt"><i>c</i></span><span style="color:000000;font-size:20pt"> = 1,...,295; the diversity (or richness) computed as the count of mammal families found in each of the 295 geographical cells that make up the USA, with: </span><latex>divc,usa=f=1NUSdf,c,usa </latex><span style="color:000000;font-size:20pt">(see Sheet 7, supplemental data appendix).</span>\n\n<span style="color:000000;font-size:20pt"><i>cw</i></span><span style="color:000000;font-size:20pt"><i><sub>f,c,usa,r=7</sub></i></span><span style="color:000000;font-size:20pt"> for  </span><span style="color:000000;font-size:20pt"><i>f </i></span><span style="color:000000;font-size:20pt">= 1,…,</span><span style="color:000000;font-size:20pt"><i>N</i></span><span style="color:000000;font-size:20pt"><i><sub>US</sub></i></span><span style="color:000000;font-size:20pt">, </span><span style="color:000000;font-size:20pt"><i>c</i></span><span style="color:000000;font-size:20pt"> = 1,...295; the cosmopolitanism weights of a family </span><span style="color:000000;font-size:20pt"><i>f</i></span><span style="color:000000;font-size:20pt"> present in a geographic cell </span><span style="color:000000;font-size:20pt"><i>c</i></span><span style="color:000000;font-size:20pt"> of the USA as measured by the count of that family presence in all seven regions of the world (not the count of presence in the 295 cells constituting the USA). Hence it is computed as </span><latex>cwf,c,usa,r=7=i=17df,i,r=7 </latex><span style="color:000000;font-size:20pt">if </span><span style="color:000000;font-size:20pt"><i>d</i></span><span style="color:000000;font-size:20pt"><i><sub>f,c,usa</sub></i></span><span style="color:000000;font-size:20pt"><sub> </sub></span><span style="color:000000;font-size:20pt">= 1</span><span style="color:000000;font-size:20pt"><sub>, </sub></span><span style="color:000000;font-size:20pt">otherwise </span><latex>cwf,c,usa,r=7=0</latex><span style="color:000000;font-size:20pt">, and where </span><span style="color:000000;font-size:20pt"><i>d</i></span><span style="color:000000;font-size:20pt"><i><sub>f,i,7</sub></i></span><span style="color:000000;font-size:20pt"> is defined earlier, albeit now restricted to the 32 families </span><span style="color:000000;font-size:20pt"><i>f</i></span><span style="color:000000;font-size:20pt"> found in the USA (</span><span style="color:000000;font-size:20pt"><i>f</i></span><span style="color:000000;font-size:20pt">=1...</span><span style="color:000000;font-size:20pt"><i>N</i></span><span style="color:000000;font-size:20pt"><i><sub>US</sub></i></span><span style="color:000000;font-size:20pt">)  (see Sheet 8, supplemental data appendix; Sheet 8 makes itself a reference to Sheet 12, which provides a slice or subset of Sheet 2–keeping only families found in the USA). </span>\n\n<span style="color:000000;font-size:20pt"><i>z</i></span><span style="color:000000;font-size:20pt"><i><sub>c,usa,r=7</sub></i></span><span style="color:000000;font-size:20pt"> is the total cosmopolitanism statistic (or zeroth moment) of a geographical cell, </span><span style="color:000000;font-size:20pt"><i>c</i></span><span style="color:000000;font-size:20pt">.  It is computed, for a </span><span style="color:000000;font-size:20pt"><i>c</i></span><span style="color:000000;font-size:20pt">, as the summation over all families </span><span style="color:000000;font-size:20pt"><i>f</i></span><span style="color:000000;font-size:20pt"> found in the USA (</span><span style="color:000000;font-size:20pt"><i>f</i></span><span style="color:000000;font-size:20pt">=1...</span><span style="color:000000;font-size:20pt"><i>N</i></span><span style="color:000000;font-size:20pt"><i><sub>US</sub></i></span><span style="color:000000;font-size:20pt">) of the cosmopolitanism weights, </span><span style="color:000000;font-size:20pt"><i>cw</i></span><span style="color:000000;font-size:20pt"><i><sub>f,c,usa,r=7</sub></i></span><span style="color:000000;font-size:20pt">, so that </span><latex>zc,usa,r=7=f=1NUScwf,c,usa,r=7=f=1NUS(i=17df,i,7if  df,c,usa= 1)</latex><span style="color:000000;font-size:20pt">.  Finally, the mean total cosmopolitanism statistic is </span><span style="color:000000;font-size:20pt"><i>z</i></span><span style="color:000000;font-size:20pt"><i><sub>c,usa,r=7</sub></i></span><span style="color:000000;font-size:20pt">/</span><span style="color:000000;font-size:20pt"><i>div</i></span><span style="color:000000;font-size:20pt"><i><sub>c,usa</sub></i></span><span style="color:000000;font-size:20pt"> (see Sheet 8, supplemental data appendix).</span>\n\n<span style="color:000000;font-size:20pt"><i>s</i></span><span style="color:000000;font-size:20pt"><i><sub>c1,c2,usa</sub></i></span><span style="color:000000;font-size:20pt">, the Smith similarity index between two geographical cells, </span><span style="color:000000;font-size:20pt"><i>c1</i></span><span style="color:000000;font-size:20pt"> and </span><span style="color:000000;font-size:20pt"><i>c2</i></span><span style="color:000000;font-size:20pt">, in the USA, with </span><span style="color:000000;font-size:20pt"><i>s</i></span><span style="color:000000;font-size:20pt"><i><sub>c1,c2,usa</sub></i></span><span style="color:000000;font-size:20pt"><i> = c</i></span><span style="color:000000;font-size:20pt"><i><sub>c1,c2,usa</sub></i></span><span style="color:000000;font-size:20pt"><sub> </sub></span><span style="color:000000;font-size:20pt">- (</span><span style="color:000000;font-size:20pt"><i>dis</i></span><span style="color:000000;font-size:20pt"><i><sub>c1,c2,usa</sub></i></span><span style="color:000000;font-size:20pt"><i> + dis</i></span><span style="color:000000;font-size:20pt"><i><sub>c2,c1,usa</sub></i></span><span style="color:000000;font-size:20pt">), where </span><span style="color:000000;font-size:20pt"><i>c</i></span><span style="color:000000;font-size:20pt"><i><sub>c1,c2,usa</sub></i></span><span style="color:000000;font-size:20pt"> is the count of common families across the pair of geographical cells, </span><span style="color:000000;font-size:20pt"><i>dis</i></span><span style="color:000000;font-size:20pt"><i><sub>c1,c2,usa</sub></i></span><span style="color:000000;font-size:20pt"><i> </i></span><span style="color:000000;font-size:20pt">is the count of families found in </span><span style="color:000000;font-size:20pt"><i>c</i></span><span style="color:000000;font-size:20pt"><i><sub>1</sub></i></span><span style="color:000000;font-size:20pt"> but not in </span><span style="color:000000;font-size:20pt"><i>c</i></span><span style="color:000000;font-size:20pt"><i><sub>2</sub></i></span><span style="color:000000;font-size:20pt"><i>, dis</i></span><span style="color:000000;font-size:20pt"><i><sub>c2,c1,usa</sub></i></span><span style="color:000000;font-size:20pt"><i> </i></span><span style="color:000000;font-size:20pt">is the count of families found in </span><span style="color:000000;font-size:20pt"><i>c</i></span><span style="color:000000;font-size:20pt"><i><sub>2</sub></i></span><span style="color:000000;font-size:20pt"> but not in </span><span style="color:000000;font-size:20pt"><i>c</i></span><span style="color:000000;font-size:20pt"><i><sub>1</sub></i></span><span style="color:000000;font-size:20pt">. In a parallel short-cut notation, we set </span><span style="color:000000;font-size:20pt"><i>s</i></span><span style="color:000000;font-size:20pt"><i><sub>c1,c2,usa</sub></i></span><span style="color:000000;font-size:20pt"> = </span><span style="color:000000;font-size:20pt"><i>a</i></span><span style="color:000000;font-size:20pt"> – (</span><span style="color:000000;font-size:20pt"><i>b</i></span><span style="color:000000;font-size:20pt"> + </span><span style="color:000000;font-size:20pt"><i>c</i></span><span style="color:000000;font-size:20pt">). Sheet 9 of the supplemental data appendix illustrates the computation of the similarity index between any geographical cell and cell</span><span style="color:010101;font-size:20pt"> #258 (covering southwestern Georgia).</span>\n\n\t\t\t\t\t\n\n\t\t\t\t\t----Image alt text---->Diagram\n\nDescription automatically generated<----media/image2.jpeg----\n\n\t\t\t\t\t<span style="color:010101"><b>Figure 2</b></span><span style="color:010101">  The arbitrarily-bounded sample study area investigated here, with 295-cell grid superimposed</span>\n\n\t\t\t\t\t\n\n\t\t\t\t\t<span style="color:010101">Figure 3 maps the familial diversities record (</span><span style="color:000000;font-size:20pt"><i>div</i></span><span style="color:000000;font-size:20pt"><i><sub>c,USA</sub></i></span><span style="color:010101">) in the 295 cells. Figure 4 maps the associated total cosmopolitanism statistics (</span><span style="color:000000;font-size:20pt">z</span><span style="color:000000;font-size:20pt"><sub>c,usa,r=7</sub></span><span style="color:010101">), and Figure 5, the calculated mean cosmopolitanism values (</span><span style="color:000000;font-size:20pt"><i>z</i></span><span style="color:000000;font-size:20pt"><i><sub>c,usa,r=7</sub></i></span><span style="color:000000;font-size:20pt">/</span><span style="color:000000;font-size:20pt"><i>div</i></span><span style="color:000000;font-size:20pt"><i><sub>c,USA</sub></i></span><span style="color:010101">). See also Sheet 14 for a R-program that generates all US maps presented in this section.)</span>\n\n\t\t\t\t\t----media/image3.jpeg----\n\n\t\t\t\t\t<span style="color:010101"><b>Figure 3</b></span><span style="color:010101">   Family-level mammalian richness (</span><span style="color:000000;font-size:20pt"><i>div</i></span><span style="color:000000;font-size:20pt"><i><sub>c,USA</sub></i></span><span style="color:010101">) across the sample study area investigated here</span>\n\n\t\t\t\t\t----media/image4.jpeg----\n\n\t\t\t\t\t<span style="color:010101"><b>Figure 4</b></span><span style="color:010101">  Total mammal family cosmopolitanism statistics (</span><span style="color:000000;font-size:20pt"><i>z</i></span><span style="color:000000;font-size:20pt"><i><sub>c,usa,r=7</sub></i></span><span style="color:010101">) across the sample study area investigated here </span>\n\n\t\t\t\t\t----media/image5.jpeg----\n\n\t\t\t\t\t<span style="color:010101"><b>Figure 5</b></span><span style="color:010101">  Mean mammal family cosmopolitanism statistics (</span><span style="color:000000;font-size:20pt"><i>z</i></span><span style="color:000000;font-size:20pt"><i><sub>c,usa,r=7</sub></i></span><span style="color:000000;font-size:20pt">/</span><span style="color:000000;font-size:20pt"><i>div</i></span><span style="color:000000;font-size:20pt"><i><sub>c,USA</sub></i></span><span style="color:010101">) across the sample study area investigated here </span>\n\n\t\t\t\t\t<span style="color:010101"> </span>\n\n\t\t\t\t\t<span style="color:010101">For our first analytical example, consider how the present model can be used to map the net </span><span style="color:010101"><i>similarities</i></span><span style="color:010101"> of single cells (or, if desired, point location tallies) in the sample area to any other one.  This has been carried out (using the Smith similarity index [3]) in Figure 6, in which the U. S.-wide mammal family faunal similarities to cell #258 (covering southwestern Georgia) are mapped. (See Sheet 9, supplemental data appendix.)  In this representation, a “track” of high similarity extends from the area around #258 westward, to eastern Texas and Oklahoma. Dissimilarity maximizes in the Great Basin area. </span>\n\n\t\t\t\t\t----media/image6.jpeg----\n\n\t<b>Figure 6</b>  Mammal family similarities with respect to cell #258 (southwestern Georgia) of the grid mapped out in Figure 2\n\n\t\n\n\tIn Figure 7 another such pattern is mapped, this one originating with the mammal families found in cell #105, located in dry north-central Nevada. (See Sheet 10, supplemental data appendix.) The falling-away of similarities to cell #105 is somewhat more regular here than in Figure 6.\n\n\t\n\n\t----media/image7.jpeg----\n\n\t<b>Figure 7  </b>Mammal family similarities with respect to cell #105 (north-central Nevada) of the grid mapped out in Figure 2\n\n\tAmong the things that stand out in Figures 3 through 7:  In Figure 3, the low diversities in the eastern Midwest (possibly a function of anthropogenic influence, plus the impact of recent glaciation) and the higher values in areas with a greater elevational/ecological diversity; in Figure 4, lower values in the Midwest and areas of “difficult” ecological conditions (southeastern California, central Nevada, and coastal Louisiana); in Figure 5, lower values in areas of heterogeneous habitat and higher values in dispersal-remote areas (southeastern United States); in Figure 6, lowest similarities in remote dry areas and a southeastern U. S. corridor effect among high similarities; in Figure 7, a more regular distance-decay effect impacted by the nearby presence of coastal Pacific forms. \n\n\tAn elaboration on the relational values mapped in Figures 6 and 7 can be created by substituting cosmopolitanism totals for the basic similarities approach applied in these (<i>i.e.</i>, presence is weighted according to the number of regions in which each form is found (<span style="color:000000;font-size:20pt"><i>cw</i></span><span style="color:000000;font-size:20pt"><i><sub>f,c,usa,r=7</sub></i></span>) The cosmopolitanism similarity values connected to the similarities mapped in Figure 7 are mapped as Figure 8. (See Sheet 11, supplemental data analysis.)\n\n\t\n\n\t\n\n\t----media/image8.jpeg----\n\n\t<b>Figure 8</b>  Mammal family similarities with respect to cell #105 (north-central Nevada) of the grid mapped out in Figure 2, weighted by cosmopolitanism values\n\n\tA regression of the Figure 7 values on the Figure 8 ones, as well as yielding a simple Pearson r correlation statistic of 0.9096, produces the set of residuals mapped as Figure 9.\n\n\t\n\n\t----media/image9.jpeg----\n\n\t<b>Figure 9</b>  Residuals from a regression of the 294 mammal total faunal cosmopolitanism similarities data of Figure 8 and the simple similarities of Figure 7\n\n\t\n\n\tIn Figure 9 the residuals from the regression are seen to be highly spatially autocorrelated; that is, the simple regression model may be regarded as invalid as is (violating the independence-of-relation-between-residuals standard), requiring that additional explanatory variables be brought to bear. In other words, the general distance-decay function of similarity between cell #105 and other cells is secondarily complex, influenced by historical and ecological forces that have caused spatially nonrandom assemblies of families of differing cosmopolitanism. \n\n\tAnother possible application of place-specific similarities scores concerns the earlier alluded-to characterization of transition zones between regions.  Treatments of this subject have produced a considerable literature of late (<i>e.g.</i>, Morrone [12‒14]; Halffter &amp; Morrone [15]; Hermogenes de Mendonça &amp; Ebach [16]).  In the present context, evaluation can proceed through comparisons of the faunas at a number of particular peripheral locations with the total diversities assigned to the (usually two, but occasionally three) adjoining regions. This will both isolate the point of transition of primary affinity, and detail its sharpness.  This is well illustrated in Figure 10, which maps part of the transition between the Holarctic and Latin American regions using the present example. In particular, for each of the 295 US geographical cells in Figure 10, we compute two Smith similarity indices, one tracking the faunal similarity of any given individual US cell with respect to the whole Holarctic region, and one tracking the faunal similarity of that same US cell with respect to the whole Latin America region. (See Sheet 13, supplemental data appendix.)\xa0These two indices can then, each, be discretized in three classes of similarity values (low, medium, high). Using principles of bivariate choropleth mapping, we obtain (3)<sup>2</sup> = 9 classes, each represented by a single color. A cell colored in a mix of dark ‘blue/red’ indicates a high score for both similarity indices while a combination of pale or light ‘blue/red’ reflects a low score for both similarity indices. This approach permits to track the transition between the Holarctic and the Latin American regions within the USA. \n\n\t\n\n\t----media/image10.jpeg----\n\n\t<b>Figure 10</b>  Mammal family faunal similarities (using the “Smith similarity index”) between individual cells of the sample study area and the overall regional mammal family tallies of the Holarctic and Latin American regions, employing a bivariate mapping technique\n\n\t\n\n\tThus, Figure 10 groups the data into pairings of three classes of values combining low/medium/high similarity to both the Holarctic and Latin American standards.  The study area here falls nearly entirely within the Holarctic region, so it will not be surprising to find that only three cells (nos. 280, 289, and 290) exhibit higher similarities to the Latin American region than to the Holarctic. These three cells are, as it turns out, right where the Holt <i>et al.</i> data place the northernmost extension of the Latin American region: southern Texas along the Rio Grande (see Figure 1). (Such placement also provides a crucial validation of the appropriateness of the Smith similarity index.) Cell nos. 198, 220, 261, and 279 (covering, respectively, expanses north and south of Los Angeles, cross-border southeasternmost Arizona, and the transnational Big Bend vicinity) are areas of relatively high family richness due largely to high environmental diversity, and demonstrate relatively large similarities to both the Holarctic and the Latin American. The six lightest-colored cells (121, northwestern Ohio; 145, central Indiana; 199, central Mohave Desert; 235, northern Mississippi; 236, northwestern Alabama; and 285 and 286, bayou Louisiana) cover areas of relatively low topographic/environmental diversity and family richness leading to low absolute similarity to both the Holarctic or Latin American regions, taken as wholes. The large expanse of pink cells (medium similarity to the Holarctic, low similarity to the Latin American) in the Eastern United States is indicative of low family richness and total cosmopolitanism there, and somewhat elevated mean cosmopolitanism (see Figures 6, 7 and 8): these conditions probably result from relative remoteness and low environmental/topographic diversity, the lagging effects of glaciation, and anthropogenic factors.\n\n\tOne can imagine various ways of extending the transition zones analysis, for example by varying bin definitions and limits, changing cell tallies by including strictly marine forms, or adding in mean cosmopolitanism statistics to aid in the depictions of the regional standards comparisons. \n\n\t\n\n<b>4. Discussion</b>\n\nThe seven-region model of mammal family distribution presented in [1] breaks with the assumption that regional classifications featuring primary units of differing size and diversity need be considered suboptimal structures; indeed, it is possible to make the argument that such systems are more likely to evolve stochastically in a manner leading to component units that are not equivalent in that sense. Nevertheless, the data in [1] clearly show that each such component in the system brings with it characteristics that are equally important to specifying conditions across the rest of the system. This reality legitimizes pattern-mapping applications making use of the statistical moments of the regionalization—in particular to the identification of standardized relations between locations and other locations (tracks, corridors, etc.)—and of those between locations and the whole, or parts thereof (cosmopolitanism comparisons, transition and boundary zones, etc.). Again, and as stated in [1], this is true whether one chooses to view the classification as a possible example of a “natural law-based” pattern, or merely an efficient phenetic device. \n\nSince the publication of [1], however, we have investigated another characteristic of world mammal faunal patterns that tends to support both the seven-region model over the other four entertained, and the “natural law” interpretation.  In all five (5- through 9-region) models, many of the <i>N</i> = 140 mammal families are distributed across two or more of their regions.  If one totals up the region-by-region diversities arrived at, a figure of <latex>t=i=1rdivi,r=269</latex> is produced for the seven-region classification (<i>r=</i>7); for the five-region (<i>r=</i>5) classification the parallel figure is 225; for six-, 260; for eight-, 293, and for nine-, 307. As a matter of straight combinatorics, one may reasonably ask what the most likely distribution of number of regional affinities might be in each case, and consider the results in view of the actual totals.  \n\nThe distribution breakdown is given by maximizing <latex>S=N!i=1agi!</latex>, where <i>N</i> is the total number of groups, <i>a</i> is the maximum group size, and <latex>gi, i=1,…,a</latex> is the number of groups with a particular size, so that, for example, if <i>t </i>= 269, the combinatorially most frequently arrived at combination of groupings is: 1 group with seven elements, 2 with six, 4 with five, 9 with four, 18 with three, 35 with two, and 70 with one.----footnote1---- The total number of groups <i>N</i> is thus 1 + 2 + 4 + 9 + 18 + 35 + 70 = 139.  It is interesting that in the context of this paper, the value for <i>N</i> is 140 (mammal families), just one off from 139, and that the number of distinct categories into which these values are distributed is seven.  For <i>t</i> = 225, 260, 293 and 307 totals, the combinatorially most frequently arrived at distribution of values also requires seven categories of groups, but the total number of groups departs further from 140, being, respectively, 115, 135, 151, and 160. Further, it makes no sense to say that in the five- and six-region classifications there can be a condition wherein a family might exist in seven regions—or, for that matter, that in the eight- and nine-region classifications it is impossible that a family can exist in that many regions. The comparison between breakdowns of combinatorial likelihood for the five different classifications and their actual totals are given in Table 1. These facts may be added to those produced in [1] in defending the seven-region model of mammal faunas over the other four models, both in terms of its greater efficiency of representation, and the possibility of its being viewed in “natural system” terms.\n\n\n\n<b>Table 1. </b><span style="font-size:20pt">Combinatoric breakdowns for the r totals for the five faunal classifications. Under the ‘max-S solution’ listings ‘1/7’, for example, may be read as “one group with seven elements”.</span>\n\n\n\n\t<span style="font-size:20pt"><b>Model</b></span>\n\n\t<span style="font-size:20pt"><b>with t=</b></span>\n\n\t<span style="font-size:20pt"><b>max-S solutions</b></span>\n\n\t<span style="font-size:20pt"><b>with total number of groups</b></span>\n\n\t\n\n\t<span style="font-size:20pt"><b>actual no. of mammal </b></span>\n\n\t<span style="font-size:20pt"><b>families present, ‘f/g’ may be read as “f families present in g regions”</b></span>\n\n\t<span style="font-size:20pt"><b>N, actual  </b></span>\n\n\t<span style="font-size:20pt"><b>families</b></span>\n\n\t<span style="font-size:20pt">5-region model</span>\n\n\t<span style="font-size:20pt">225</span>\n\n\t<span style="font-size:20pt">1/7 2/6 3/5 7/4 15/3 30/2 58/1   </span>\n\n\t<span style="font-size:20pt">116</span>\n\n\t<span style="font-size:20pt">2/5 5/4 13/3 36/2 84/1</span>\n\n\t<span style="font-size:20pt">140</span>\n\n\t<span style="font-size:20pt">6-region model</span>\n\n\t<span style="font-size:20pt">260</span>\n\n\t<span style="font-size:20pt">1/7 2/6 4/5 8/4 17/3 35/2 68/1</span>\n\n\t<span style="font-size:20pt">135</span>\n\n\t<span style="font-size:20pt">6/5 12/4 11/3 38/2 73/1</span>\n\n\t<span style="font-size:20pt">140</span>\n\n\t<span style="font-size:20pt">7-region model</span>\n\n\t<span style="font-size:20pt">269</span>\n\n\t<span style="font-size:20pt">1/7 2/6 4/5 9/4 18/3 35/2 70/1</span>\n\n\t<span style="font-size:20pt">139</span>\n\n\t<span style="font-size:20pt">3/6 6/5 10/4 12/3 36/2 73/1   </span>\n\n\t<span style="font-size:20pt">140</span>\n\n\t<span style="font-size:20pt">8-region model</span>\n\n\t<span style="font-size:20pt">293</span>\n\n\t<span style="font-size:20pt">1/7 2/6 5/5 10/4 19/3 38/2 76/1</span>\n\n\t<span style="font-size:20pt">151</span>\n\n\t<span style="font-size:20pt">3/7 3/6 9/5 6/4 17/3 32/2 70/1   </span>\n\n\t<span style="font-size:20pt">140</span>\n\n\t<span style="font-size:20pt">9-region model</span>\n\n\t<span style="font-size:20pt">307</span>\n\n\t<span style="font-size:20pt">1/7 2/6 5/5 10/4 20/3 41/2 81/1</span>\n\n\t<span style="font-size:20pt">160</span>\n\n\t<span style="font-size:20pt">3/8 0/7 6/6 9/5 7/4 13/3 33/2 69/1</span>\n\n\t<span style="font-size:20pt">140</span>\n\n\n\nIf evolution on this planet is in fact, generally speaking, a stochastically-enacted process, it should not be surprising that elements of its spatial expression should follow along in response. These elements not only reflect phylogeny <i>per se</i>, but the way the earth system serves its spatial/physical expression through the operation of geological and ecological constraints and opportunities.\n\n<b>Supplementary Materials</b>: The following supporting information can be downloaded at \n\nData Appendix._second_paper_FINAL.xlsx\n\n\n\n<span style="color:000000"><b>Author contribution:</b></span>\n\n<span style="color:000000">Charles Smith conceived the project, wrote the main manuscript text, and contributed the data leading to the analyses and tables and figures; Patrick Georges contributed the mathematical </span>modeling and statistics <span style="color:000000">nomenclature</span>. <span style="color:000000">Ngoc Nguyen </span>contributed all <span style="color:000000">figures</span> (using R) and programmed the combinatory question underlying the ‘Discussion’ section (using Python)<span style="color:000000">.\xa0T</span><span style="color:201F1E">he supplementary materials were prepared by Patrick Georges and Ngoc Nguyen.</span><span style="color:000000">\xa0All\xa0authors reviewed the manuscript.\xa0</span>\n\n\n\n<b>References</b>\n\n[1] Smith, C.H.; Georges, P.; Nguyen, N. Evolution and Biogeography, and the Systems Measurement of Mammalian Biotas. <i>Life</i> <b>2023</b>, 13, 873.\n\n[2] Holt, B.G.; Lessard, J.P.; Borregaard, M.K.; Fritz, S.A.; Araújo, M.B.; Dimitrov, D.; Fabre, P.H.; Graham, C.H.; Graves, G.R.; Jønsson, K.A.; et al. An Update of Wallace’s Zoogeographic Regions of the World. <i>Science</i> <b>2013</b>, 339, 74–78. [<u> https://doi.org/10.1126/science.1228282</u> \n\n[3] Smith, C.H. A System of World Mammal Faunal Regions. I. Logical and Statistical Derivation of the Regions. <i>J. Biogeogr.</i> <b>1983</b>, 10, 455–466. <u>https://doi.org/10.2307/2844752</u>\n\n[4] Smith, C.H. A System of World Mammal Faunal Regions. II. The Distance Decay Effect upon Inter-regional Affinities. <i>J. Biogeogr.</i> <b>1983</b>, 10, 467–482. <u>https://doi.org/10.2307/2844605</u>\n\n[5] Wilson, A.G. <i>Entropy in Urban and Regional Modelling</i>; Pion: London, UK,1970.\n\n[6] Whittaker, R.J.; Riddle, B.R.; Hawkins, B.A.; Ladle, R.J. The Geographical Distribution of Life and the Problem of Regionalization: 100 Years after Alfred Russel Wallace. <i>J. Biogeogr.</i> <b>2013</b>, 40, 2209–2214. <u>https://doi.org/10.1111/jbi.12235</u> \n\n[7] Wilson, A.G. Boltzmann, Lotka and Volterra and Spatial Structural Evolution: an Integrated Methodology for some Dynamical Systems. <i>J. R. Soc. Interface</i> <b>2008</b>, 5, 865–871. <u>https://doi.org/10.1098/rsif.2007.1288</u> \n\n[8] Smith, C.H. A General Approach to the Study of Spatial Systems. I. The Relational Representation of Measurable Attributes. <i>Int. J. Gen. Syst.</i> <b>1986</b>, 12, 359–384. <u>https://doi.org/10.1080/03081078608934943</u>\n\n[9] Smith, C.H.; Derr, M. “In Space” or “as Space”?: a New Model. <i>Life</i> <b>2012</b>, 2, 243–254. <u>https://doi.org/10.3390/life2030243</u> \n\n[10] Smith, C.H. Areographic Representation of Faunal Characteristics through a “Second order” Relational Approach. <i>Evol. Theory</i> <b>1983</b>, 6, 225–232. <u>https://digitalcommons.wku.edu/dlps_fac_pub/41</u> \n\n[11] <span style="color:222222">Wilson, D.E.; Reeder, D.\xa0</span><span style="color:222222"><i>Mammal Species of the World: A Taxonomic and Geographic Reference</i></span><span style="color:222222">, 3rd ed.; John Hopkins University Press: Baltimore, MD, USA, 2005.</span>\n\n[12] Morrone, J. Biogeographic Areas and Transition Zones of Latin America and the Caribbean Islands Based on Panbiogeographic and Cladistic Analyses of the Entomofauna. <i>Ann. Rev. Entomol.</i> <b>2006</b>, 51, 467–494. <a href="https://doi.org/10.1146/annurev.ento.50.071803.130447"><span style="color:auto">https://doi.org/10.1146/annurev.ento.50.071803.130447</span></a> \n\n[13] Morrone, J. Biogeographical Transition Zones: a Search for Conceptual Synthesis. <i>Biol. J. Linn. Soc.</i> <b>2010</b>, 113, 1–12. <a href="https://doi.org/10.1111/bij.12333"><span style="color:auto">https://doi.org/10.1111/bij.12333</span></a>\n\n[14] Morrone, J. <i>The Mexican Transition Zone</i>. Springer: Cham, Switzerland, 2020.\n\n[15] Halffter, G.; Morrone, J. An Analytical Review of Halffter’s Mexican Transition Zone, and its Relevance for Evolutionary Biogeography, Ecology and Biogeographical Regionalization. <i>Zootaxa</i> <b>2017,</b> <u>HTTPS://DOI.ORG/10.11646/ZOOTAXA.4226.1.1 </u>\n\n[16] Hermogenes de Mendonça, L.; Ebach, M.C. A Review of Transition Zones in Biogeographical Classification. <i>Biol. J. Linn. Soc.</i> <b>2020</b>, 131, 717–736. <u>https://doi.org/10.1093/biolinnean/blaa120</u>\n\n15\n\n\n\n\n\n\n\nfootnote1)\t This combinatory question and results computed herein have been programmed in Python.  \n\n\n\n